README.md

```diff
@@ -290,6 +290,7 @@ If you are not a programmer but would like to contribute, check out the [Awesome
 - [tree-sitter-legesher-python](https://github.com/legesher/tree-sitter-legesher-python) _(label: Good-First-Issue)_ <br> Learn and code in Python using your native language.
 - [wemake-python-styleguide](https://github.com/wemake-services/wemake-python-styleguide) _(label: level:starter)_ <br> The strictest and most opinionated python linter ever!
 - [Zulip](https://github.com/zulip/zulip) _(label: good-first-issue)_ <br> Powerful open source group chat.
+- [Trulens](https://github.com/truera/trulens) (label: Begineer-Friendly)_ <br> TruLens-Eval and TruLens-Explain are housed in separate packages and can be used independently. Also it Evaluates your llm performance.
```

Comment:
TruLens is an open-source package that provides instrumentation and evaluation tools for large language model (LLM) based applications.